meta/meta.go

```diff
@@ -517,3 +518,19 @@ func (m *Meta) GetHistoryDDLJob(id int64) (*model.Job, error) {
 	err = job.Decode(value)
 	return job, errors.Trace(err)
 }
+
+// IsBootstrapped returns whether we have already run bootstrap or not.
+// return true means we don't need doing any other bootstrap.
+func (m *Meta) IsBootstrapped() (bool, error) {
+	value, err := m.txn.GetInt64(mBootstrapKey)
```

Comment:
For set "1", we can use GetInt64 to fetch this number directly, no need to do another conversation.